grumphp.yml

```diff
@@ -61,7 +61,7 @@ parameters:
             clover_file: build/logs/clover.xml
             level: 99
         infection:
-            threads: 10
+            threads: 4
```

Comment:
You tests are using very little RAM, keeping this at 10 threads should work as it seems.

Reply:
Ok, good to know! I mostly do that so my laptop doesn't get stuck when running the tests :)